doc/cascadia/profiles.schema.json

```diff
@@ -535,4 +536,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
I never know which line endings we actually want. So I'm just gonna tag it here to point it out.